### PR TITLE
DOCS: Add reference documentation for inspector views

### PR DESCRIPTION
**Pull request by grebaldi**
Hi there,

this PR adds a full reference documentation for every built-in inspector view. Documentation for the `ui.inspector.views` key has also been added to the "NodeType Definition Reference".

I targeted 7.3, because I was unsure about how to treat documentation in regards to our LTS promises, and thought it makes sense to have the documentation available for all current Neos versions, because it works for all of them. It likely works for earlier versions as well.

## TODOS

- [x] Add reference documentation for `Neos.Neos/Inspector/Views/Data/ColumnView`
- [x] Add reference documentation for `Neos.Neos/Inspector/Views/Data/TableView`
- [x] Add reference documentation for `Neos.Neos/Inspector/Views/Data/TimeSeriesView`
- [x] Add reference documentation for `Neos.Neos/Inspector/Views/NodeInfoView`
- [x] Document inspector views in "NodeType Definition Reference"

### CODE REVIEW

**Review by kdambekalns — APPROVED** [review]
Awesome, thanks!

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+                  label: 'this month'
+                -
+                  data: thisMonth.growthPercent
+                  label: '% growth'
+    # ...
+
+``viewOptions`` Reference:
+
+``dataSource`` (required string)
+  Class name of class implementing ``Neos\Neos\Service\DataSource\DataSourceInterface``
```
Comment (mhsdesign) [inline Neos.Neos/Documentation/References/InspectorViewsReference/index.rst]:
```suggestion
  Identifier for a DataSource

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+                  data: uniqueVisitors
+                  suffix: ' visitors'
+                -
+                  data: percentage
+                  suffix: '%'
+    # ...
+
+``viewOptions`` Reference:
+
+``dataSource`` (required string)
```
Comment (mhsdesign) [inline Neos.Neos/Documentation/References/InspectorViewsReference/index.rst:221]: also here ;)

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+    ui:
+      inspector:
+        views:
+          timeSeriesViewExample:
+            label: 'TimeSeriesView Example'
+            group: examples
+            view: 'Neos.Neos/Inspector/Views/Data/TimeSeriesView'
+            viewOptions:
+              subtitle: 'last month'
+              dataSource: vendor-site-time-series-view
```
Comment (mhsdesign) [inline Neos.Neos/Documentation/References/InspectorViewsReference/index.rst:323]: And here ;)

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+                timeData: date
+                valueData: uniqueVisitors
+              chart:
+                selectedInterval: weeks
+                yAxisFromZero: false
+    # ...
+
+``viewOptions`` Reference:
+
+``dataSource`` (required string)
```
Comment (mhsdesign) [inline Neos.Neos/Documentation/References/InspectorViewsReference/index.rst:335]: Aaand here :D

**Review by mhsdesign — CHANGES_REQUESTED** [review]
Sooo many thanks @grebaldi for writing this lovely documentation!!!! ❤️ (you even have alt tags for images?!!)

I noticed that your description of the dataSource argument in the nodeType doesn’t match the actual one…

Also fyi I think the DataSource interface is a joke as I think Neos relies on the AbstractDataSource ;P
But I find it much more problematic that data sources are so haaard to test :P but that’s for another time ^^

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst (hunk 3 above)
Comment (kdambekalns) [reply Neos.Neos/Documentation/References/InspectorViewsReference/index.rst:323]: Here it's fine, that is just an identifier…

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+                  suffix: ' visitors'
+                -
+                  data: percentage
+                  suffix: '%'
+    # ...
+
+``viewOptions`` Reference:
+
+``dataSource`` (required string)
+  Class name of class implementing ``Neos\Neos\Service\DataSource\DataSourceInterface``
```
Comment (kdambekalns) [inline Neos.Neos/Documentation/References/InspectorViewsReference/index.rst]:
```suggestion
  Identifier of a DataSource (given in the implementation class)
```

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst (hunk 1 above)
Comment (kdambekalns) [reply Neos.Neos/Documentation/References/InspectorViewsReference/index.rst]:
```suggestion
  Identifier of a DataSource (given in the implementation class)
```

File: Neos.Neos/Documentation/References/InspectorViewsReference/index.rst [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+                valueData: uniqueVisitors
+              chart:
+                selectedInterval: weeks
+                yAxisFromZero: false
+    # ...
+
+``viewOptions`` Reference:
+
+``dataSource`` (required string)
+  Class name of class implementing ``Neos\Neos\Service\DataSource\DataSourceInterface``
```
Comment (kdambekalns) [inline Neos.Neos/Documentation/References/InspectorViewsReference/index.rst]:
```suggestion
  Identifier of a DataSource (given in the implementation class)
```

**Review by kdambekalns — CHANGES_REQUESTED** [review]

**Review by kdambekalns — APPROVED** [review]

**Review by mhsdesign — APPROVED** [review]

**Pull request merged**